web/js/MapUI/Components/CreateMap/CreateMap.js

```diff
@@ -0,0 +1,278 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  forOwn as lodashForOwn,
+  debounce as lodashDebounce,
+} from 'lodash';
+import OlMap from 'ol/Map';
+import OlView from 'ol/View';
+import OlKinetic from 'ol/Kinetic';
+import OlControlScaleLine from 'ol/control/ScaleLine';
+import { altKeyOnly } from 'ol/events/condition';
+import OlInteractionPinchRotate from 'ol/interaction/PinchRotate';
+import OlInteractionDragRotate from 'ol/interaction/DragRotate';
+import OlInteractionDoubleClickZoom from 'ol/interaction/DoubleClickZoom';
+import OlInteractionPinchZoom from 'ol/interaction/PinchZoom';
+import OlInteractionDragPan from 'ol/interaction/DragPan';
+import OlInteractionMouseWheelZoom from 'ol/interaction/MouseWheelZoom';
+import OlInteractionDragZoom from 'ol/interaction/DragZoom';
+import * as olProj from 'ol/proj';
+import util from '../../../util/util';
+import {
+  refreshRotation,
+  updateRenderedState,
+  updateMapUI,
+} from '../../../modules/map/actions';
+import { saveRotation } from '../../../map/util';
+import {
+  MAP_DISABLE_CLICK_ZOOM,
+  MAP_ENABLE_CLICK_ZOOM,
+  MAP_DRAG,
+  MAP_MOVE_START,
+  MAP_ZOOMING,
+} from '../../../util/constants';
+import { startLoading, stopLoading, MAP_LOADING } from '../../../modules/loading/actions';
+import { granuleFootprint } from '../../../map/granule/util';
+
+const { events } = util;
+
+const CreateMap = (props) => {
+  const {
```

Comment:
startLoading, stopLoading, updateMapUI, & updateRenderedState are previously declared (lines 35, 25 & 24). Which declaration should we keep?

Reply:
These are more Redux actions where I just use the same names.